docker/1.4.0/final/Dockerfile.gpu.serving

```diff
@@ -0,0 +1,35 @@
+ARG py_version
+
+FROM awsdeeplearningteam/mxnet-model-server:base-gpu-py$py_version
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache mxnet-cu92mkl==1.4.0.post0 \
+                           keras-mxnet==2.2.4.1 \
```

Comment:
same comment as before

Reply:
spoke with someone from the mxnet team. keras-mxnet has support for inference. so we will leave it in.